pymongo/compression_support.py

```diff
@@ -123,16 +122,10 @@ class ZlibContext(object):
     compressor_id = 2
 
     def __init__(self, level):
-        self.compress: Callable[[bytes], bytes]
-
-        # Jython zlib.compress doesn't support -1
-        if level == -1:
-            self.compress = zlib.compress
-        # Jython zlib.compress also doesn't support 0
-        elif level == 0:
-            self.compress = _zlib_no_compress
-        else:
-            self.compresss = lambda data, _: zlib.compress(data, level)
```

Comment:
`compresss` -> `compress` was the bug fix. I also removed the Jython workarounds.